tasktypes/vendoring.py

```diff
@@ -66,12 +67,17 @@ def process_task(self, library, task):
         # regardless of if outgoing commits exist or not.
         self.logger.log("Removing any outgoing commits before moving on.", level=LogLevel.Info)
 
-        self.cmdProvider.run(["hg", "status"])  # hey what the fruck?
-        ret = self.cmdProvider.run(["hg", "strip", "roots(outgoing())", "--no-backup"], clean_return=False)
-        if ret.returncode == 255:
-            if "abort: empty revision set" not in ret.stderr.decode():
-                self.logger.log("hg strip failed but not in a permissible way", level=LogLevel.Warning)
-                ret.check_returncode()
+        # If we are on TC, update to the HEAD commit to avoid stripping WIP commits on holly
+        self.cmdProvider.run(["hg", "status"])
```

Comment:
Thank you for removing my Battlestar Galactica level curse word. If you want we can remove this entire line as I originally only put it here for debugging.

Reply:
Yeah; I think it's a good idea to keep it there :)